tools/donate-cpu.py

```diff
@@ -45,6 +45,10 @@
         stop_time = arg[-5:]
         print('Stop time:' + stop_time)
     elif arg.startswith('-j'):
+        if not re.match(r'-j\d+', arg):
```

Comment:
Should the regex be `r'-j ?\d+'` to handle one space after `-j`?

Reply:
No, while Cppcheck would accept this, the script does not work when there is whitespace after '-j'. First I had implemented it to accept whitespace, but tests and looking closer at the code have shown that it is not valid. Whitespace in fact is interpreted as the end of the argument. Using whitespace could have been the issue of the useless results that were uploaded yesterday.